java/maven/nbproject/project.properties

```diff
@@ -22,7 +22,7 @@ javadoc.apichanges=${basedir}/apichanges.xml
 javadoc.arch=${basedir}/arch.xml
 javahelp.hs=maven.hs
 extra.module.files=maven-nblib/
-spec.version.base: 2.156
+spec.version.base: 2.157
```

Comment:
As far as I can see, this should actually be?

`spec.version.base=2.157.0`

Use of colon means it gets skipped in increment task, and if not skipped needs to end with `.0`?

Reply:
@neilcsmith-net - I am confused; so should I rebase against `delivery` plus adjust spec numbers to fit into `delivery` codebase ?

Reply:
@sdedic yes, it's confusing!  To get into NB17 it needs to be based on top of `delivery`.  Normally, spec versions are fixed from branch, because the first thing we do on `master` at branch is a spec version increment (eg. #5312 )

However, the `maven` (and `java.editor.lib`) modules use a colon rather than equals in the properties file, and are being missed out of that increment.  I've only just noticed that.

IMO, unless there's a reason those modules shouldn't be incrementing every release, we should change this to what I suggested in my previous comment, and keep your API comments as is, when rebasing on top of delivery.